projects/plugins/jetpack/modules/sharedaddy/sharing-service.php

```diff
@@ -939,12 +937,12 @@ function sharing_process_requests() {
 }
 
 /**
- * Gets the url to customise the sharing buttons in Calypso.
+ * Gets the url to customise the sharing buttons WP-Admin.
```

Comment:
Minor grammar mistake
```suggestion
 * Gets the url to customise the sharing buttons in WP-Admin.
```